tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -896,45 +896,46 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 	// 5. Create CNS register volume with above created FCD, AccessMode as "ReadOnlyMany".
 	// 6. verify  the error message.
 	// 7. Delete Resource quota.
-	ginkgo.It("[csi-supervisor] Verify static provisioning when AccessMode is ReadWriteMany or ReadOnlyMany", func() {
-		var err error
-		ctx, cancel := context.WithCancel(context.Background())
-		defer cancel()
+	ginkgo.It("[csi-supervisor-parallelized] Verify static provisioning when AccessMode is ReadWriteMany or ReadOnlyMany",
+		func() {
+			var err error
```

Comment:
remove this indentation change

Reply:
Without this indentation, the line will hit the maximum characters in the line